common.neon

```diff
@@ -5,17 +5,20 @@ services:
 			- addExtraModule(@Mathematicator\Engine\SampleModule)
 	mathematicator.queryNormalizer:
 		factory: Mathematicator\Engine\QueryNormalizer
-	mathematicator.translator:
-		factory: Mathematicator\Engine\Translator
 	- Mathematicator\NumberRewriter
 	- Mathematicator\NaturalTextFormatter
 	- Mathematicator\Engine\SampleModule
+	- Mathematicator\Router\Router
 	- Mathematicator\Engine\Controller\OtherController
 	- Mathematicator\Engine\Controller\ErrorTooLongController
+	- Mathematicator\Engine\Translation\TranslatorHelper
+	- Symfony\Component\Translation\Translator('cs_CZ')
```

Comment:
Default locale can be loaded here: https://github.com/baraja-core/localization/blob/master/src/Localization.php#L133